spec/spec_helper.rb

```diff
@@ -1,7 +1,4 @@
-if ENV['TRAVIS']
-  require 'codeclimate-test-reporter'
-  CodeClimate::TestReporter.start
-elsif ENV['COVERAGE']
+if ENV['COVERAGE']
```

Comment:
Do we need to configure any environment variables for CI? Looks like the coverage was not found:

```
$ bundle exec codeclimate-test-reporter
Coverage results not found
The command "bundle exec codeclimate-test-reporter" exited with 1.
```